tools/composefs-from-json.c

```diff
@@ -667,6 +673,8 @@ int main(int argc, char **argv)
 	cleanup_free FILE **input_files = NULL;
 	bool no_sandbox = false;
 
+	tzset();
```

Comment:
shouldn't we enforce `TZ=GMT` before?

Reply:
I don't think we need that, as we use timegm(). I'm honestly not sure we need tzset() either...